subprojects/gradle/runner/device-provider/src/main/kotlin/com/avito/android/runner/devices/internal/kubernetes/KubernetesApi.kt

```diff
@@ -15,7 +16,7 @@ internal interface KubernetesApi {
     suspend fun getPodDescription(podName: String): String
     suspend fun deleteDeployment(deploymentName: String)
     suspend fun createDeployment(deployment: Deployment)
-    suspend fun getPods(deploymentName: String): List<Pod>
+    suspend fun getPods(deploymentName: String): Result<List<Pod>>
```

Comment:
Return Failure, not an empty list
It will help consumer react clearer

Reply:
What about other methods? Should they also return Result?
E.g. getPod

Reply:
I think they should but it's not critical